server/model.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
     return db.Table.findAll({ where: query });
   },
 
-  // makeReservations: (restaurantId, tableId) => {
-
-  // },
+  makeReservations: (query) => db.Table.findOne({ where: query })
+    .then((table) => table.update({ is_open: false }))
```

Comment:
what happens if the table isn't open? Should we make sure the table is available before setting `is_open: false`?

Reply:
good point, i may add this check in the future, but for now i intend to only display to the client, open tables. In this way, the client cannot select a table that is already taken to make a reservation for.